fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraFixity.java

```diff
@@ -114,4 +139,23 @@ protected Session session() {
     protected String externalPath() {
         return externalPath;
     }
+
+    private String setUpBaseURL(final UriInfo uriInfo) {
+        try {
+            final URI baseURL = uriInfo.getBaseUri();
+            LOGGER.debug("setting baseURL = " + baseURL.toString());
+            return baseURL.toString();
+        } catch (Exception ex) {
+            LOGGER.warn("Error setting baseURL", ex);
+        }
+        return "";
+    }
+
+    private String setUpAgent(final HttpHeaders headers) {
+        if (!StringUtils.isBlank(headers.getHeaderString("user-agent"))) {
+            return headers.getHeaderString("user-agent");
```

Comment:
Is this the "user" we want for the fixity check? The "user-agent" header contains information like:
"Mozilla Firefox" or "curl"